test/components/previews/admin/forms/container_component_preview.rb

```diff
@@ -0,0 +1,62 @@
+module Admin
+  module Forms
+    class ContainerComponentPreview < ViewComponent::Preview
+      # Default Form Container
+      # ---------------
+      #
+      # @param title text
+      # @param description textarea
+      def default(title: "title", description: "description")
+        render Admin::Forms::ContainerComponent.new(title, description)
+      end
+
+      # Form Container with aside
+      # ---------------
+      #
+      # @param title text
+      # @param description textarea
+      def with_aside(title: "title", description: "description")
+        render_with_template(locals: {
+          title: title,
+          description: description
+        })
```

Comment:
@joemasilotti I'd like to listen what do you think about this. In this preview file I'm using preview templates because otherwise render components inside content or slots is a little bit different than the normal use case. Example:

Rendering component from preview:
```ruby
def with_aside(title: "title", description: "description")
  render Admin::Forms::ContainerComponent.new(title, description) do |component|
    component.with_aside do
      component.render Admin::BadgeComponent.new("active")
    end
  end
end
```

This works perfectly but in the documentation we'll see that `component.render` that is not what we normally use and maybe can generate confusion. I don't have a strong preference here, so what do you think/prefer?

Reply:
Good catch, thanks for calling this out. Why can't we use `component.render` in a preview?